src/native/components/abSpecific/FourDigitInputComponent.js

```diff
@@ -91,6 +91,9 @@ class FourDigitInputComponent extends Component {
     })
   }
   onBlur () {
+    if (this.props.dontForceFocus) {
+      return
+    }
     this.inputRef.focus()
```

Comment:
So, the gist of this commit is that this code is basically disabled. The `this.state.isFocused` variable is never read anywhere, and you are now handling the white color with your outer container component, so the only remaining change is that `this.inputRef.focus()` won't happen now. Yup, it makes sense that we wouldn't re-focus in an `onBlur` handler.

Seems solid, but maybe we can just delete `this.state.isFocused` next week when we are less regression-focused.